src/codeu/chat/server/View.java

```diff
@@ -45,14 +45,23 @@
 
   private final Model model;
 
+  private static final ServerInfo info = new ServerInfo();
+
   public View(Model model) {
     this.model = model;
   }
-  
+
   // This is implemented to avoid lots of checks with BasicView objects that should be views
   // If the BasicView object is not an instance of a view then it should throw a UnsupportedOperationException.
+
+  @Override
+  public ServerInfo getInfo() {
+    //Accesses ServerInfo, better method for a server info object
+    return info;
+  }
+
   public ServerInfo getInfo(){
-    throws new UnsupportedOperationException("A view that is a BasicView but not a view object cannot use getInfo in this version.");
+    throw new UnsupportedOperationException("A view that is a BasicView but not a view object cannot use getInfo in this version.");
```

Comment:
This clashes with the method above.

This will not compile. Do you mean to change the behavior here? Your exception message indicates that BasicView does not support this operation, but this class is not BasicView.